### ISSUE TITLE
Documentation with issue (or not?)

### ISSUE THREAD

**Issue by nrsantos** [aaronpowell/db.js#74]
Hi,

On your example you have:
```
server.people.query( 'indexName' )
        .bound( 'answer', 30, 50 )
        .execute()
        .done( function ( results ) {
            //do something with the results
        } );
```
Shouldn't be :
```
server.people.query( 'answer' )
        .bound(  30, 50 )
        .execute()
        .done( function ( results ) {
            //do something with the results
        } );
```

?

**Comment by brettz9** [comment]
I've fixed this in PR #134

**Comment by brettz9** [comment]
With the PR #134 now being merged, I think this issue can be closed.
